descriptors/ModuleDescriptor-template.json

```diff
@@ -614,7 +614,7 @@
       "description": "Create consortia configuration"
     },
     {
-      "permissionName": "consortia.inventory.share.local.instance",
+      "permissionName": "consortia.inventory.local.sharing-instances.execute",
```

Comment:
Is it okay to change a permission name without using a "replaces" functionality? https://github.com/folio-org/okapi/blob/master/doc/guide.md#permissions-and-the-_tenantpermissions-interface